golang/go-guestbook/src/backend/Dockerfile

```diff
@@ -20,11 +20,10 @@ RUN go build -o /app/backend .
 # container and point it to the "dlv debug" command:
 #   * UNCOMMENT the following ENTRYPOINT statement,
 #   * COMMENT OUT the last ENTRYPOINT statement
-# Start the "dlv debug" server on port 3000 of the container. (Note that the
-# application process will NOT start until the debugger is attached.)
-# ENTRYPOINT ["dlv", "debug", ".",  "--api-version=2", "--headless", "--listen=:3000", "--log"]
+# Start the "dlv debug" server on port 3000 of the container.
+ENTRYPOINT ["dlv", "exec", "/app/backend", "--continue", "--accept-multiclient", "--api-version=2", "--headless", "--listen=:3000", "--log"]
```

Comment:
Was the debugger experience improved in the last few months? I remember we deliberately decided to not enable go debugger by default because there were multiple issues with it.

Reply:
Yes, I've worked with the vscode-go extension to improve the debugging experience so user can now start the program without waiting for the debugger to be attached.

Reply:
sg, CI/CD succeeded, so I'm merging this